initialization/config.py

```diff
@@ -15,15 +15,15 @@ def init_config(app):
     if app_env == 'development':
         app.config.update(
             DEBUG=True,
-            ASSETS_DEBUG=True,
+            ASSETS_DEBUG=False,
```

Comment:
The "debug" mode for assets loads them one at a time, but this slows down page loads on development and it's not that useful.